### DISCUSSION TITLE
Channels capacity

### DISCUSSION THREAD

**Discussion by Darth-Coin** [ACINQ/phoenix#365] [Q&A] [answered]
Hi,
I was doing some tests, trying to refill some channels.
I thought that Phoenix is using MPP and could receive a payment into all channels, distributing the sats in the best way could find a way.
But is not.
Not just that is not spliting the payment in multiple shards and refill all channels available but is not even able to receive a decent amount in each, if I just sent a smaller payment.
According to the channels list I have these 4 channels.
![image](https://github.com/ACINQ/phoenix/assets/56997124/0449dd51-000b-4fe3-ab5e-d44b055d78d9)

So according to this list, I should have:
chan A can receive: 10370 sats
chan B can receive 9620 sats
chan B can receive 10328 sats
chan D can receive 10435 sats

I also checked in channels details for how much is the channel reserve and I saw that is a surprisingly only 546 sats reserve for each channel! How is possible only that?
But OK, even if I would deduct that reserve, I should be able to send at least 10000 sats for those 3 channels and 9000 sats for the other one. Not talking about sending the total of 40753 sats that it should be available to receive.

I tried from 4 different LN wallets, even sending 1000 sats per payment and was not able to receive anymore. In condition that is displaying 10000 sats available to receive in at least 3 channels.

Please, can you explain me how is the logic of these channels?
I am trying to use the maximum space in these channels but the payment always fail now, whatever amount I use to pay.
All channels are active and well.

EDIT: I disabled deliberately the option in settings that "open channels on the fly". I wanted to TEST a full refill of existing channels. Or at least to find a maximum limit could be refilled, even that will be done, partially one by one sending to each channel.
I think that from a total capacity (displayed) of 183335 sats a 40753 sats is quite a lot of "reserve" that cannot be used. This is like 30% wasted space!

Or Phoenix wanted to push users to open more new channels and charge that 3000 sats fee for each one?

**Comment by dpad85** [comment, marked as answer]
Hello,

Phoenix is indeed using MPP but the split won't be optimal in many cases. If the sender is not using trampoline (which is the case with many LN wallets) the payment will likely not be split. The sender does not know about your 4 channels A, B, C, and D.

Also, a channel's incoming capacity is affected by the on-chain feerate. The feerate is currently quite high, and if your channels are small (like in your screenshot) this can lock a substantial portion of your incoming capacity. But that is not wasted space, it guarantees the safety of your channels.

You can find more info about this in the FAQ: https://phoenix.acinq.co/faq#how-much-can-i-receive-with-my-existing-channels

**Comment by Darth-Coin** [comment]
Thanks for answering @dpad85 ,

I know about commit fees that could be variable depending on the onchain fees (especially in case of small channels).
That was first thing that came into my mind when I saw that payments can't get through.
I was surprised to not see this displayed in some way in Phoenix - channels.
User must be informed about these aspects in a more transparent way.
I myself, I am quite experienced in running LN nodes and I understand these aspects. But please think about a total new user that do not know these technical aspects. Will be easier and clearer for the user is is displayed in some way (at least with approximation) that commit fee reserve. So the user will know that is time to open a new channel or open bigger ones.

OK now is clear, were the commit fees affecting it.
